XNAControls/XNAPanel.cs

```diff
@@ -100,6 +122,9 @@ public override void Update(GameTime gameTime)
     {
         Alpha += AlphaRate * (float)(gameTime.ElapsedGameTime.TotalMilliseconds / 100.0);
 
+        if (BackgroundAnimation != null)
+            BackgroundTexture = BackgroundAnimation.Next(gameTime) ?? BackgroundTexture;
```

Comment:
Once the changes to `Animation` have been done, this can be reworked into

```suggestion
        if (BackgroundAnimation != null)
        {
            BackgroundAnimation.Update(gameTime);
            BackgroundTexture = BackgroundAnimation.CurrentFrame;
        }
```

Reply:
Done by 3f835bdb71f947a79333e3352c5974c9f36d4490.